beets/library.py

```diff
@@ -418,6 +418,14 @@ class Item(LibModel):
         'lyricist':             types.STRING,
         'composer':             types.STRING,
         'composer_sort':        types.STRING,
+        'parent_work':          types.STRING,
+        'parent_work_disambig': types.STRING,
+        'work':                 types.STRING,
+        'work_disambig':        types.STRING,
+        'performer':            types.STRING,
+        'performer_sort':       types.STRING,
+        'parent_composer':      types.STRING,
+        'parent_composer_sort': types.STRING,
```

Comment:
It actually shouldn't be necessary to add these fields explicitly in library.py. You can just use flexible attributes, which get created transparently when you add data to Items.